packages/enhanced/src/lib/container/runtime/FederationRuntimePlugin.ts

```diff
@@ -46,23 +46,29 @@ class FederationRuntimePlugin {
 
   static getTemplate(runtimePlugins: string[], bundlerRuntimePath?: string) {
     // internal runtime plugin
+    const normalizedBundlerRuntimePath = (
+      bundlerRuntimePath || BundlerRuntimePath
+    ).replaceAll('\\', '/');
```

Comment:
Can you use path.normalize here?

Reply:
yeah can we look at that? otherwise theres also a normalize webpack path function in this codebase it  think, for us to resolve the right webpack.

@2heal1 is that right or should we implement this?

Reply:
We did try a path.normalize in the   normalizWebpackPath util but it apparently was not working, I'll have another look

Reply:
I've just double checked, normalizeWebpackPath isn't used for these paths, they're @module-federation modules not webpack.

path.normalize considers the path as already being valid so makes no changes. We tried some different methods but this is the only way we could generate paths, that work on Windows, and the way that the webpack path is currently being extracted.

Reply:
gotcha

Reply:
yes, i miss to test the function in Windows, much appreciated for modifying this @jonthomp !

And i need to test the modify can be worked as expect in our codebase. I will test it today.

Reply:
@2heal1 happy to help!

Sorry I also missed the remaining Windows fix that we have in a patch. I have now added it to this PR.

On Windows, the line `webpackLocationWithDetail.split(':').slice(0, -2)[0]` returns `C` because the path is split into `['C', 'some\path\to\webpack']`